web/src/components/vis/MetaboliteTable.vue

```diff
@@ -28,42 +43,30 @@ export default {
     },
   },
 
-  data() {
-    return {
-      format: format('.4e'),
-      pagination: {
-        rowsPerPage: -1,
-      },
-      // need to use this workaround, since `header-cell` as slot name doesn't work
-      // and `headerCell` isn't allowed by the linter
-      headercell: 'headerCell',
-      colors,
-    };
-  },
-
-  computed: {
-    selectedItems: {
+  setup(props, { emit }) {
+    const format = ref(d3Format('.4e'));
+    const pagination = reactive({ rowsPerPage: -1 });
+    const headercell = ref('headerCell');
```

Comment:
See the removed comment about why this was used.  That comment applied to the pug linter.  You can remove this ref and use the `headerCell` or `header-cell` slot name literal in the template below.